### PR TITLE
[Gui] Edit/Placement not active when no object selected

### PR DESCRIPTION
**Pull request by 0penBrain**
- [X]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [X]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [X]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [X]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [X]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [X]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the [FreeCAD bug tracker issue number](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) in case a particular commit solves or is related to an existing issue on the tracker. Ex: `Draft: fix typos - fixes #0004805`

### CODE REVIEW

File: src/Gui/CommandDoc.cpp [hunk 1]
```diff
@@ -1574,7 +1574,7 @@ void StdCmdPlacement::activated(int iMsg)
 
 bool StdCmdPlacement::isActive(void)
 {
-    return (Gui::Control().activeDialog()==0);
+    return (Selection().getCompleteSelection().size() > 0) || (Gui::Control().activeDialog() != 0);
```
Comment (yorikvanhavre) [inline src/Gui/CommandDoc.cpp:1577]: So the command is active if (selection > 0 OR active dialog open)... shouldn't it be (selection > 0 AND no active dialog)?

Comment (0penBrain) [reply src/Gui/CommandDoc.cpp:1577]: Indeed I was too fast on this and this is inappropriate. Fortunately you saw it.
Should be better now. I applied same rule as Std_TransformManip.
There could be still improved filtering on this, but it already brings some improvement and consistency.

Comment (0penBrain) [reply src/Gui/CommandDoc.cpp]: @yorikvanhavre Actually last time I messed up my git commands and it was not fixed. Now is OK (for sure). :)

**Comment by yorikvanhavre** [comment]
Thanks for the fix! This lets the command active even if another task dialog is running... Did you check if that's OK? (The Edit/Placement command is able to close the former task dialog)?

**Comment by 0penBrain** [comment]
@yorikvanhavre yes that works, it closes the actual dialog and opens the new one. Anyway this is a good remark. Is that still what we really want? Or do you think it's better if the command is disabled if a dialog is already open?

**Comment by yorikvanhavre** [comment]
Ok, thanks for checking! I have no strong opinion on this, I was just thinking of staying in line with what was currently implemented, there might have been a reason why the IsActive method was made that way... But it might as well have simply been copy/pasted from another command.

**Pull request merged**